ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/ProxyTest.kt

```diff
@@ -25,4 +29,22 @@ class ProxyTest : ClientLoader() {
             assertEquals("proxy", response)
         }
     }
+
+    @Test
+    fun testProxyWithSerialization() = clientTests(listOf("Js")) {
+        config {
+            engine {
+                proxy = ProxyBuilder.http(HTTP_PROXY_SERVER)
+            }
+
+            install(JsonFeature)
+        }
+
+        test { client ->
+            val response = client.get<ProxyResponse>("http://google.com/json")
```

Comment:
Isn't it better to not specify google but some non-existing domain since we expect it to not requested anyway?

Reply:
To get wrong response in case of failure